client/client-multi/client-c/src/kaa/kaa_user.c

```diff
@@ -250,6 +250,8 @@ kaa_error_t kaa_user_request_serialize(kaa_user_manager_t *self, kaa_platform_me
         writer->current += sizeof(uint16_t);
         *((uint16_t *) writer->current) = KAA_HTONS((uint16_t) self->user_info->user_verifier_token_len);
         writer->current += sizeof(uint32_t); /* verifier token length + reserved */
+        KAA_LOG_TRACE(self->logger, KAA_ERR_NONE, "Serializing external system authentication parameters: user external id length %u, virifier length %u"
```

Comment:
Please, print out external_id, access_token and verifier_token instead of length